### PR TITLE
Minor Fixes and Cleanup

### PR DESCRIPTION
**Pull request by alexjhawk**
This PR is primarily for minor fixes and cleanup, which I had outstanding on my system.

~~The `Fix default config file` change is the only commit that would warrant an immediate release, but I planned to replace the existing file on the current release. A new, full release is unnecessary because existing connector installations with the improper default value would likely be 'upgraded,' and the configuration file would remain unchanged/'unfixed.'~~

### CODE REVIEW

File: src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java [hunk 1]
```diff
@@ -84,6 +84,27 @@ public class CanaryApiRequestBuilder {
    */
   private static final String JSON_KEY_FILE_SIZE = "fileSize";
 
+  /**
+   * The JSON key for the auto write no data field.
+   *
+   * @since 1.0.0
```
Comment (TomKimsey) [inline src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java:90]: 1.0.3?

File: src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java [hunk 2]
```diff
@@ -84,6 +84,27 @@ public class CanaryApiRequestBuilder {
...
+   */
+  private static final String JSON_KEY_AUTO_WRITE_NO_DATA = "autoWriteNoData";
+
+  /**
+   * The JSON key for the extend data field.
+   *
+   * @since 1.0.0
```
Comment (TomKimsey) [inline src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java:97]: 1.0.3?

File: src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java [hunk 3]
```diff
@@ -84,6 +84,27 @@ public class CanaryApiRequestBuilder {
...
+   */
+  private static final String JSON_KEY_EXTEND_DATA = "extendData";
+
+  /**
+   * The JSON key for the insert replace data field.
+   *
+   * @since 1.0.0
```
Comment (TomKimsey) [inline src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java:104]: 1.0.3?

File: web-docs/docs/02-CHANGELOG.mdx [hunk 4]
```diff
@@ -26,7 +37,8 @@ toc_max_heading_level: 2
 ### Bug Fixes
 - N/A
 ### Other
-- Migrated temporary abstract package to SC extensions library. There are no functional changes to the connector.
+- Migrated temporary abstract package to SC extensions library. There are no functional changes to
```
Comment (TomKimsey) [inline web-docs/docs/02-CHANGELOG.mdx:40]: This change should not be in the release commit

**Review by TomKimsey — CHANGES_REQUESTED** [review]

File: src/main/java/com/hms_networks/sc/canary/CanaryConnectorConfig.java [hunk 5]
```diff
@@ -574,13 +574,13 @@ public int getApiClientFileSize() {
    * @since 1.0.0
    */
   public boolean getApiClientAutoCreateDatasets() {
-    boolean apiClientTimeout = DEFAULT_CONFIG_API_AUTO_CREATE_DATASETS;
+    boolean apiClientAutoCreateDatasets = DEFAULT_CONFIG_API_AUTO_CREATE_DATASETS;
```
Comment (it-hms) [inline src/main/java/com/hms_networks/sc/canary/CanaryConnectorConfig.java:717]: Please also update comment.

**Review by it-hms — CHANGES_REQUESTED** [review]

File: src/main/java/com/hms_networks/sc/canary/CanaryConnectorConfig.java (hunk 5 above)
Comment (alexjhawk) [reply src/main/java/com/hms_networks/sc/canary/CanaryConnectorConfig.java:717]: Updated

File: web-docs/docs/02-CHANGELOG.mdx (hunk 4 above)
Comment (alexjhawk) [reply web-docs/docs/02-CHANGELOG.mdx:40]: Moved to its own commit

File: src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java (hunk 3 above)
Comment (alexjhawk) [reply src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java]: Changed

File: src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java (hunk 2 above)
Comment (alexjhawk) [reply src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java]: Changed

File: src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java (hunk 1 above)
Comment (alexjhawk) [reply src/main/java/com/hms_networks/sc/canary/api/CanaryApiRequestBuilder.java]: Changed

**Review by it-hms — APPROVED** [review]

**Review by TomKimsey — APPROVED** [review]

**Pull request merged**